src/main/java/net/snowflake/ingest/streaming/SnowflakeStreamingIngestChannel.java

```diff
@@ -247,6 +262,8 @@ public interface SnowflakeStreamingIngestChannel {
    */
   InsertValidationResponse insertRow(Map<String, Object> row, @Nullable String offsetToken);
 
+  InsertValidationResponse insertRow(Map<String, Object> row);
```

Comment:
in which cases will a customers use insertRow without an offsetToken?

Reply:
Yeah shouldn't this be ` InsertValidationResponse insertRow(Map<String, Object> row, @Nullable String offsetToken);`?

Reply:
remember that we say the offset_token will be per row for Rowset API?